newIDE/app/src/guidelines/index.js

```diff
@@ -0,0 +1,214 @@
+      };
+    });
+  };
+
+  componentDidUpdate(prevProps, prevState) {
+    if (this.state.index !== prevState.index) {
+      this._inputRef.current.updatePosition();
+
+      setTimeout(() => this.setState({ indexData: this.state.index }), 300);
+    }
+  }
+
+  render() {
+    const { opened } = this.props;
+
+    let nextOrFinish;
+    let imageTutorial;
+    let closeOrWikipage;
+
+    if (this.state.index === this.state.indexMax) {
+      nextOrFinish = (
+        <Button
+          variant="contained"
+          color="primary"
+          onClick={this.togglePopover}
+        >
+          Finish
+        </Button>
+      );
+    } else {
+      nextOrFinish = (
+        <Button
+          variant="contained"
+          color="primary"
+          disabled={this.state.index === this.state.indexMax ? true : false}
+          onClick={this.next}
+        >
+          Next
+        </Button>
+      );
+    }
+
+    if (this.state.index === this.state.indexMax) {
+      closeOrWikipage = (
+        <Button
+          variant="contained"
+          onClick={() => {
+            Window.openExternalURL(getHelpLink('/tutorials'));
+            this.togglePopover();
+          }}
+        >
+          See wikipage
+        </Button>
+      );
+    } else {
+      closeOrWikipage = (
+        <Button variant="contained" onClick={this.togglePopover}>
+          Close
+        </Button>
+      );
+    }
+
+    if (guidelines[this.state.indexData].imageSource !== undefined) {
+      imageTutorial = (
+        <img
+          class="fit-picture"
+          src={guidelines[this.state.indexData].imageSource}
+          alt={guidelines[this.state.indexData].imageAlt}
+        />
+      );
+    } else {
+      imageTutorial = () => {
+        return;
+      };
+    }
+
+    return (
+      <div className="app">
+        <Button
+          variant="contained"
+          disabled={this.state.open}
+          onClick={this.togglePopover}
+        >
+          Open popover
+        </Button>
+        <Popover
+          action={this._inputRef}
+          open={this.state.open}
+          anchorReference="anchorPosition"
+          anchorPosition={{
+            top: guidelines[this.state.index].position.x,
+            left: guidelines[this.state.index].position.y,
+          }}
+          anchorOrigin={{
+            vertical: 'top',
+            horizontal: 'left',
+          }}
+          transformOrigin={{
+            vertical: 'top',
+            horizontal: 'left',
+          }}
+        >
+          <Grid container style={styles.container}>
+            <Grid item xs={12} style={styles.description}>
+              <Typography variant="h5">
+                {guidelines[this.state.indexData].title}
+              </Typography>
+              <Typography wrap="true">
+                {guidelines[this.state.indexData].description}
+              </Typography>
+              <br />
+              {imageTutorial}
+            </Grid>
+            <Grid
```

Comment:
You might want to take a look at `Line` and `Column` from `UI/Grid` - it's a basic grid system used almost everywhere in the UI. Should be enough for 90% of what we need. Don't spend too much time adapting your stuff for now though - we can always easily rebuild the UI of your popovers later.